src/Npgsql/NpgsqlTracingOptions.cs

```diff
@@ -6,4 +9,14 @@ namespace Npgsql;
 /// </summary>
 public class NpgsqlTracingOptions
 {
+    /// <summary>
+    /// Gets or sets an action to enrich an Activity.
+    /// </summary>
+    /// <remarks>
+    /// <para><see cref="Activity"/>: the activity being enriched.</para>
+    /// <para>string: the name of the event.</para>
+    /// <para>object: the raw object from which additional information can be extracted to enrich the activity.
+    /// The type of this object depends on the event, which is given by the above parameter.</para>
+    /// </remarks>
```

Comment:
We should link to https://www.npgsql.org/doc/diagnostics/tracing.html here, with `<see href>`, we may even want to omit the inline explanations (better for users to see the docs?).